src/sparseml/pytorch/utils/verbosity_helper.py

```diff
@@ -0,0 +1,35 @@
+# Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from enum import Enum
+
+
+__all__ = [
+    "Verbosity",
+]
+
+
+class Verbosity(Enum):
+    DEFAULT = 1
```

Comment:
making these integers isn't going to be easy for anyone but advanced users to know which levels are which. Let's switch these over to string constants so the yaml mapping matches what the user is trying to input. Additionally should support non truthy such as False and None and have that evaluate to OFF